app/api/client/create/route.ts

```diff
@@ -0,0 +1,78 @@
+import dbConnect from "@/app/lib/dbConnect";
+import mongoose from "mongoose";
+import Client from "@/app/models/Client";
+import { ObjectId } from "mongoose";
+import bcrypt from "bcrypt";
+
+interface ClientModelInterface {
+  _id: string;
```

Comment:
Change` _id: string;` → `_id: Types.ObjectId;`

The reason for this is because `_id`, when received from a model is not actually a `string`, its an `ObjectId`